### PR TITLE
BBCode img height and alt are missing #4068

### PR DESCRIPTION
**Pull request by xillibit**
Pull Request for Issue #4068 .

### CODE REVIEW

**Comment by 810** [comment]
we need some placeholders on the input of the modal, so users can see what he need to enter

**Comment by 810** [comment]
and size doesn't work

**Comment by xillibit** [comment]
> we need some placeholders on the input of the modal, so users can see what he need to enter

I will add some placeholders

> and size doesn't work

What do-you mean by it doesn't work ?

File: src/components/com_kunena/template/crypsis/layouts/bbcode/image/default.php [hunk 1]
```diff
@@ -26,14 +27,15 @@
 $attributesLink = $config->lightbox ? ' class="fancybox-button" rel="fancybox-button"' : '';
 $attributesImg  = ' style="max-height: ' . (int) $config->imageheight . 'px;"';
 $attributesImg .= $size ? ' width="' . (int) $size . '"' : '';
+$attributesImg .= $alt ? ' alt="' . $alt . '"' : '';
 ?>
```
Comment (810) [inline src/components/com_kunena/template/crypsis/layouts/bbcode/image/default.php:31]: change to:

```
$width = $size ? (int) $size . "px;"  : 'auto ';
$attributesImg  = ' style="max-height: ' . (int) $config->imageheight . 'px;' . ' max-width:' . $width . '"';
```

**Comment by xillibit** [comment]
Updated

**Comment by sshcli** [comment]
/components/com_kunena/language/en-GB/en-GB.com_kunena.ini
Line 408

File: src/components/com_kunena/language/en-GB/en-GB.com_kunena.ini [hunk 2]
```diff
@@ -402,6 +402,10 @@ COM_KUNENA_EDITOR_MODAL_TITLE_CODE_SETTINGS="Code Settings"
 COM_KUNENA_EDITOR_MODAL_TITLE_PICTURE_SETTINGS="Picture Settings"
 COM_KUNENA_EDITOR_MODAL_TITLE_PICTURE_SETTINGS_SIZE="Size"
 COM_KUNENA_EDITOR_MODAL_TITLE_PICTURE_SETTINGS_URL="URL"
+COM_KUNENA_EDITOR_MODAL_TITLE_PICTURE_SETTINGS_ALT="Alt"
+COM_KUNENA_EDITOR_MODAL_TITLE_PICTURE_SETTINGS_SIZE_PLACEHOLDER="Enter the Size of the image"
+COM_KUNENA_EDITOR_MODAL_TITLE_PICTURE_SETTINGS_URL_PLACEHOLDER="Enter the URL of the image"
+COM_KUNENA_EDITOR_MODAL_TITLE_PICTURE_SETTINGS_ALT_PLACEHOLDER="Eenter the Alt of the image"
```
Comment (810) [inline src/components/com_kunena/language/en-GB/en-GB.com_kunena.ini]: typo

File: src/components/com_kunena/language/en-GB/en-GB.com_kunena.ini [hunk 3]
```diff
@@ -402,6 +402,10 @@ COM_KUNENA_EDITOR_MODAL_TITLE_CODE_SETTINGS="Code Settings"
 COM_KUNENA_EDITOR_MODAL_TITLE_PICTURE_SETTINGS="Picture Settings"
 COM_KUNENA_EDITOR_MODAL_TITLE_PICTURE_SETTINGS_SIZE="Size"
 COM_KUNENA_EDITOR_MODAL_TITLE_PICTURE_SETTINGS_URL="URL"
+COM_KUNENA_EDITOR_MODAL_TITLE_PICTURE_SETTINGS_ALT="Alt"
+COM_KUNENA_EDITOR_MODAL_TITLE_PICTURE_SETTINGS_SIZE_PLACEHOLDER="Enter the Size of the image"
```
Comment (810) [inline src/components/com_kunena/language/en-GB/en-GB.com_kunena.ini]: Enter the width of the image

**Pull request merged**